README.md

```diff
@@ -3,53 +3,62 @@
 
 # Azure IoT Edge LoRaWAN Starter Kit
 
-Experimental sample implementation of LoRaWAN components to connect LoRaWAN antenna gateway running IoT Edge directly with Azure IoT.
-
-The goal of the project is to provide guidance and a reference for Azure IoT Edge users to experiment with LoRaWAN technology.
-
-## Background
-
-LoRaWAN is a type of wireless wide-area networking that is designed to allow long-range communication at a low bit rate among low-power connected objects, such as sensors operated on a battery.
-
-Network topology is of star-of-stars type, with the leaf sensors sending data to gateways for forwarding telemetry to and receiving commands from backing Internet services. Nowadays, even for simple scenarios like having 10 devices connected to a single LoRaWan gateway (hardware with antenna), you need to connect your gateway to a Network Server and then work through connectors provided by the server vendor to integrate your LoRa gateways and devices with the back end. These setups can be connected to Azure IoT Hub quite easily. As a matter of fact [such scenarios exist](https://github.com/loriot/AzureSolutionTemplate). Customers looking for an operated network with national or international reach (e.g. fleet operators, logistics) will tend to choose this setup accepting the potentially higher complexity and dependency on the network operator.
-
-However, customers looking for any of the following are expected to prefer a setup where the LoRaWAN network servers runs directly on the gateway/Azure IoT Edge:
-
-- Primarily coverage on their own ground (e.g. manufacturing plants, smart buildings, facilities, ports).
-- Capabilities that Azure IoT edge brings to the table:
-  - Local processing on the gateway.
-  - Offline capabilities of the gateway.
-  - Gateway management.
-- Homogenous management of devices and gateways independent of connectivity technology.
-
-## Functionality
+The LoRaWAN starter kit is an *experimental* cross platform private network implementation of the [LoRaWAN specification]('https://lora-alliance.org/resource-hub/lorawantm-specification-v102'). It enables users to setup their own LoRaWAN network that can connect to LoRa based nodes (sensors) and send decoded message packets to Azure IoT Hub for cloud based processing, analytics and other workloads. Alternatively, it allows sending commands from the cloud to the end nodes. The goal of the the project is to provide guidance and a reference for Azure IoT Edge users to experiment with LoRaWAN technology.
```

Comment:
Without this information "Experimental sample implementation of LoRaWAN components to connect LoRaWAN antenna gateway running IoT Edge directly with Azure IoT." I think it might be hard for people to understand what this project exactly implements.

Reply:
i moved the above statement as the description of the git hub page, does it make it clear?

Reply:
Looks good to me now.